pkg/chains/formats/intotoite6/intotoite6.go

```diff
@@ -286,5 +289,18 @@ func gitInfo(tr *v1beta1.TaskRun) (commit string, url string) {
 			url = r.Value
 		}
 	}
+
+	url = spdxGit(url, "")
 	return
 }
+
+// supports the SPDX format which is recommended by in-toto
+// ref: https://spdx.dev/spdx-specification-21-web-version/
```

Comment:
This link is more targeted in the doc (but maybe more likely to break?) https://spdx.dev/spdx-specification-21-web-version/#h.49x2ik5